tutorials/graph-hello-world/graph-hello-world.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Hello Graph! Write Your First SAP Graph Application
+description: Get a brief introduction to SAP Graph, including the SAP Graph Navigator and programming basics.
+auto_validation: true
+time: 20
+tags: [ tutorial>beginner, software-product>sap-business-technology-platform]
+primary_tag: software-product>sap-integration-suite
+---
+
+## Details
+### You will learn
+  - What SAP Graph entails
+  - How to navigate connected data entities in the SAP Graph Navigator
+  - `hello world` programming essentials to try out SAP Graph
+
+  SAP Graph is a new unified API for SAP. It addresses run-away API complexity and integration challenges by providing developers a single connected and unified view of all their business data, consolidating the data models of data sources like SAP S/4HANA, SAP Sales Cloud and SAP SuccessFactors into one, unified and connected, data model, representing all the data in a landscape. We call this the Business Data Graph.
+
+  !![SAP Graph Overview](SAP-Graph-overview_sm.png)
+
+  A data graph is a connected graph, with data entities serving as the nodes of the graph and edges that represent relationships (which we call associations) between those entities. The nodes of the Business Data Graph are the common enterprise business objects, like Customer, Supplier, Product, Sales and Purchase Order, General Ledger, etc.
```

Comment:
Who is "we"? Use You and not the royal We.(unless, in rare cases, you are referring to SAP).

Reply:
Rephrased.